LambdaS3FileZipper/Aws/S3FileRetriever.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text.RegularExpressions;
```

Comment:
🙈 can we standardize on an indentation spacing here?  Maybe we should turn on some sort of style cop system in this repo...